unittests/InstructionCountCI/Secondary.json

```diff
@@ -265,462 +265,388 @@
       ]
     },
     "cmovo ax, bx": {
-      "ExpectedInstructionCount": 8,
+      "ExpectedInstructionCount": 5,
       "Optimal": "No",
       "Comment": "0x0f 0x40",
       "ExpectedArm64ASM": [
-        "uxth w20, w7",
-        "uxth w21, w4",
-        "ldr w22, [x28, #728]",
-        "ubfx w23, w22, #28, #1",
-        "cmp x23, #0x0 (0)",
-        "csel w20, w20, w21, ne",
-        "bfxil x4, x20, #0, #16",
-        "str w22, [x28, #728]"
+        "ldr w20, [x28, #728]",
+        "ubfx w20, w20, #28, #1",
+        "cmp w20, #0x0 (0)",
+        "csel w20, w7, w4, ne",
+        "bfxil x4, x20, #0, #16"
```

Comment:
Is this bfxil possible to deal with somehow?

Reply:
Sadly this bfxil can't be removed since 16-bit operations insert in to the GPR instead of zero extending.

Reply:
Right, ok... that's indeed unfortunate but nothing to be done I suppose